app/src/main/res/values/strings.xml

```diff
@@ -462,21 +462,21 @@
 
     <string name="settings_notification_heading">Choose notification action</string>
     <string name="settings_notification_desc">Action buttons to show with the RethinkDNS notification: %1$s</string>
-    <string name="settings_notification_desc1">Pause / Stop (default)</string>
-    <string name="settings_notification_desc2">DNS / DNS + Firewall</string>
+    <string name="settings_notification_desc1">Pause / stop (default)</string>
```

Comment:
I like "thing/other thing", but didn't change any.
Most of them are part of the greater "+" or "Plus" consideration too, so it is a bit confusing.
Some plus signs I moved to observe the "one space either side" paradigm.

Reply:
gotcha.

uppercase `S` is preferred since `Stop` is text that's also used for the `Stop` action button.

Reply:
There is no uppercase behind the slash elsewhere because it isn't correct.
It would indicate a new sentence on a new line.